### DISCUSSION TITLE
Is there any way to change split direction using state?

### DISCUSSION THREAD

**Discussion by notshriram** [johnwalley/allotment#443] [Q&A]
When I try to toggle the `vertical` property through a state, the layout just breaks. It works fine on initial page load when vertical is true and when vertical is false, however when I try to switch vertical from true to false or the other way round, it just breaks the whole layout..

**Comment by johnwalley** [comment]
Hi @notshriram. Thanks for taking the time to ask the question. It's come up quite a lot recently (see https://github.com/johnwalley/allotment/issues/457) and I do intend to add support for changing orientation. I just need to find the time!

You can track progress with this issue: https://github.com/johnwalley/allotment/issues/341.

Thanks again for using the library and I appreciate it's annoying that it breaks when changing the vertical prop 😬
